### PR TITLE
Touch up changes.

### PR DESCRIPTION
**Pull request by ostreifel**

### CODE REVIEW

File: scripts/app.ts [hunk 1]
```diff
@@ -18,4 +18,7 @@ var provider = () => {
     }
 };
 
-VSS.register("mariamclaughlin.hitcount-control-dev.hitcount-control-contribution", provider);
\ No newline at end of file
+
+const publisherId = VSS.getExtensionContext().publisherId;
+const extensionId = VSS.getExtensionContext().extensionId;
+VSS.register(`${publisherId}.${extensionId}.hitcount-control-contribution`, provider);
```
Comment (sferg-msft) [inline scripts/app.ts]: This can be just this:
VSS.register(VSS.getContribution().id, provider);

File: vss-extension.json [hunk 2]
```diff
@@ -8,40 +8,37 @@
...
         }
     },
     "links": {
-        "home": {
-            "uri": "https://bit.ly"
-        },
-        "getstarted": {
-            "uri": "https://bit.ly"
+        "support": {
+            "uri": "mailto:witiq@microsoft.com"
```
Comment (sferg-msft) [inline vss-extension.json]: this should link to this GitHub repo

File: scripts/app.ts (hunk 1 above)
Comment (ostreifel) [inline scripts/app.ts]: Updatd

File: vss-extension.json (hunk 2 above)
Comment (ostreifel) [inline vss-extension.json]: Updated

**Pull request merged**